src/style.css

```
@@ -0,0 +1,19 @@
+@import url(./variables.css);
+@import url(./card.css);
+@import url(./stars.css);
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 198px;
```

```suggestion
  width: 200px;
```
and add `box-sizing: border-box;` to `*`. also check `width` in other places